gatsby-cli missing the `better-opn` dependency

<!--
  Please fill out each section below, otherwise your issue will be closed. This info allows Gatsby maintainers to diagnose (and fix!) your issue as quickly as possible.

  Useful Links:
  - Documentation: https://www.gatsbyjs.org/docs/
  - How to File an Issue: https://www.gatsbyjs.org/contributing/how-to-file-an-issue/

  Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
-->

## Description

Error when running `gatsby new`.  I believe the dependency is missing but is required [here](https://github.com/gatsbyjs/gatsby/blob/99fb7b44810d658805211073ddfc18e508b57c87/packages/gatsby-cli/src/init-starter.js#L13).

### Steps to reproduce

run `gatsby new`

### Expected result

a new project should be created

### Actual result

Get this error in the cli: `Error: Cannot find module 'better-opn'`

### Environment

Run `gatsby info --clipboard` in your project directory and paste the output here.

Same, was just getting ready to open an issue.  

I run `nvm`, and had the issue with both `10.16.0` and `11.13.0`

This from a fresh installation of `gatsby-cli` this morning.

----

```shell
$ gatsby new my-default-starter https://github.com/gatsbyjs/gatsby-starter-default

/Users/morgan/.nvm/versions/node/v11.13.0/lib/node_modules/gatsby-cli/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js:53
        throw ex;
        ^

Error: Cannot find module 'better-opn'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:655:15)
    at Function.Module._load (internal/modules/cjs/loader.js:580:25)
    at Module.require (internal/modules/cjs/loader.js:711:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.<anonymous> (/Users/morgan/.nvm/versions/node/v11.13.0/lib/node_modules/gatsby-cli/lib/init-starter.js:32:13)
    at Module._compile (internal/modules/cjs/loader.js:805:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:816:10)
    at Module.load (internal/modules/cjs/loader.js:672:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:612:12)
    at Function.Module._load (internal/modules/cjs/loader.js:604:3)
```

--- a/packages/gatsby-cli/package.json
+++ b/packages/gatsby-cli/package.json
@@ -12,6 +12,7 @@
   "dependencies": {
     "@babel/code-frame": "^7.0.0",
     "@babel/runtime": "^7.0.0",
+    "better-opn": "^0.1.4",
     "bluebird": "^3.5.0",
     "chalk": "^2.4.2",
     "ci-info": "^2.0.0",

